### DISCUSSION TITLE
AI藏偏見？！

### DISCUSSION THREAD

**Discussion by zixunn** [1101-GS4719-C/Python-Community-GS4719-C#320] [General]
AI也存在著傳統偏見，不知可以如何避免，畢竟人們只會越來越仰賴AI人工智慧
[數位科技崛起更出現新一波的不平等](https://news.ltn.com.tw/news/life/breakingnews/3696439)

**Comment by cody910120** [comment]
希望AI可以中立一點就好，我不想看到藍綠新聞戰

**Comment by chun1203** [comment]
畢竟AI學習的數據就是真實的數據，要如何避免AI產生偏見是讓人值得思考的問題

**Comment by 20010906** [comment]
沒想到AI在學習數據時，也會學習到存在於人類間的偏見與歧視。

**Comment by Joan0518** [comment]
最廣泛討論的偏見類型發生在數據收集期間，深度學習算法只能從我們提供給它分析的數據中學習，如果訓練數據將偏見的資料輸入演算法，演算法自然也一定會輸出偏見。
也可以參考這篇文章~~
[【TECH】AI將改變世界　但「AI偏見」亦會
](https://www.hk01.com/%E4%B8%96%E7%95%8C%E5%B0%88%E9%A1%8C/507244/tech-ai%E5%B0%87%E6%94%B9%E8%AE%8A%E4%B8%96%E7%95%8C-%E4%BD%86-ai%E5%81%8F%E8%A6%8B-%E4%BA%A6%E6%9C%83)

**Reply by JAIMEHUNG** [reply]
我覺得最後一段講得很好，正是因人們對事情的看法有偏見才會產生AI偏見。那也可讓我們以此去思考審視自己。

**Comment by yt7l0** [comment]
AI是根據人類給的資料去學習，真正有偏見的應該是人類吧

**Comment by 109302013evantsao** [comment]
很多AI都是由大量資料所組成的，整體資料有可能會往特定方向傾倒，故我認為產生偏見是正常的

**Comment by pekoranoinu** [comment]
AI是由大數據組合而成，有偏見都是人造成的。

**Comment by chupochun** [comment]
AI畢竟是人類創造出來的 應該有辦法修改這類偏見

**Comment by ghost** [comment]
正因為是人類所作，偏見越難以完美地彌平，但大數據又是AI背後運算的來源，只能說從前提來說真的是勢必會發生的結果
